package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rg-bot",
-  "version": "1.7.0",
+  "version": "1.8.0",
```

Comment:
I realize we keep increasing the minor version instead of the patch version - I feel like we are making so many updates rn that we should be increasing the patch version. i.e. 1.7.1

Reply:
this one, probably could make it a patch version, but most of them we changed and/or added apis, which meant to me that minor was fine / correct